sdk/ml/azure-ai-ml/azure/ai/ml/constants/_common.py

```diff
@@ -577,3 +577,8 @@ class IdentityType:
     AML_TOKEN = "aml_token"
     USER_IDENTITY = "user_identity"
     MANAGED_IDENTITY = "managed_identity"
+
+class JobTypes:
```

Comment:
Consider reusing from here https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ml/azure-ai-ml/azure/ai/ml/constants/_job/job.py#:~:text=class%20JobType(,%3D%20%22spark%22

Reply:
We should 100% reuse that class. If we are trying to say "component deployment creation only supports these X job types" then we can have a separate array or Set of those specific job types instead